packages/aws-cdk-lib/aws-events/lib/event-pattern.ts

```diff
@@ -109,8 +116,45 @@ export class Match implements IResolvable {
   /**
    * Matches any string that doesn't start with the given prefix.
    */
-  public static anythingButPrefix(prefix: string): string[] {
-    return this.fromObjects([{ 'anything-but': { prefix: prefix } }]);
+  public static anythingButPrefix(...values: string[]): string[] {
```

Comment:
Changing an existing argument from a single `string` to `string[]` is a breaking change.
If we implement this, we would need to use feature flags, which would increase the complexity of the PR.
Please either avoid breaking changes as much as possible or modify the PR to use feature flags.

Example PRs:
- https://github.com/aws/aws-cdk/pull/31639
- https://github.com/aws/aws-cdk/pull/31475

Reply:
Is it a breaking change? If the function is called with a single value like `anythingButPrefix('foo')` then it still works as before, so it _is_ backwards compatible.

Otherwise I can make it something like this:

```ts
public static anythingButPrefix(prefix: string, ...values: string[]): string[] {
  values.unshift(prefix);

  // rest of code
}
```

Reply:
Another option would be to use function overloading:

```ts
public static anythingButPrefix(prefix: string): string[];
public static anythingButPrefix(...values: string[]): string[] {
  return this.anythingButConjunction('prefix', values);
}
```

Reply:
I'm really sorry for my misunderstandings.. This is not a breaking change and you don't need to update any more.

Reply:
No problem, thanks for taking the time to review.

What are the next step? Should I merge/rebase the latest changes from the `main` branch?